userspace/libscap/emscripten/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+# note: since macOS is effectively non-existent, this library will go away
+# as soon as we clean up the interface enough
+add_library(scap_platform scap_procs.c scap_ppm_sc.c)
```

Comment:
If affirmative, i'd wait until Grzeg's work is merged before going with this one, since he has been rebasing his PRs for months now :)

Reply:
I need to check. Which of his PR concern is on top of your mind? These are simple stubs, so I'm happy to move them according to the new design or eventually help @gnosek integrate it in his patches in case we want to merge this first.

Reply:
1/n (https://github.com/falcosecurity/libs/pull/1040) should be (mostly) fine since it's a no-op but merging only that is pointless
2/n (https://github.com/falcosecurity/libs/pull/1041) will almost certainly conflict
3/n (https://github.com/falcosecurity/libs/pull/1042) might be fine but since it builds on top of 2/n, it won't apply cleanly either

BTW, from a drive-by look, it seems `emscripten` will fit right in in the platform work (either as a separate platform, or possibly as a generic platform instance that does not list processes, users etc.)

Reply:
@gnosek do you prefer to merge yours first and then let us adapt this one accordingly, or the contrary?

Reply:
Does the order imply *who* will do the adapting? ;D

Reply:
@gnosek I'm ok in merging yours first then :)